tough-kms/src/lib.rs

```diff
@@ -0,0 +1,139 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT OR Apache-2.0
```

Comment:
We should add these (or some subset), which I copied from tough/lib.rs. These will enforce certain lints. The most important ones are `#![deny(rust_2018_idioms)]` and `#![deny(missing_docs)]`. Once you add these you will have compilation errors that you have to fix.

```suggestion
// SPDX-License-Identifier: MIT OR Apache-2.0

#![forbid(missing_debug_implementations, missing_copy_implementations)]
#![deny(rust_2018_idioms)]
// missing_docs is on its own line to make it easy to comment out when making changes.
#![deny(missing_docs)]
#![warn(clippy::pedantic)]
#![allow(
    clippy::module_name_repetitions,
    clippy::must_use_candidate,
    clippy::missing_errors_doc
)]
```